src/pages/home/report/ReportActionCompose.js

```diff
@@ -66,18 +72,24 @@ class ReportActionCompose extends React.Component {
         this.setIsFocused = this.setIsFocused.bind(this);
         this.focus = this.focus.bind(this);
         this.comment = props.comment;
+        this.autoFocusEnabled = getPlatform() === 'desktop' || (getPlatform() === 'web' && !canUseTouchScreen());
 
         this.state = {
-            isFocused: true,
+            isFocused: this.autoFocusEnabled,
             textInputShouldClear: false,
             isCommentEmpty: props.comment.length === 0,
             isMenuVisible: false,
         };
     }
 
     componentDidUpdate(prevProps) {
-        // When any modal goes from visible to hidden, bring focus to the compose field
-        if (prevProps.modal.isVisible && !this.props.modal.isVisible) {
+        // We want to manually set the focus only when.
+        //  1) Auto focus is enabled(we are on desktop/Desktop Web).
+        //  1) Screen is focsued.
+        //  2) When any modal goes from visible to hidden &
+        if (this.autoFocusEnabled && this.props.isScreenFocused
```

Comment:
`autoFocusEnabled` is maybe better named something like `this.shouldFocusInputOnScreenFocus`. Since we may be focusing in other contexts in the future and the focus isn't "disabled" per se it's just not what should happen when the parent screen gets focus.